.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Fixes #[Add issue number here]
+
+  Changes:
+<!-- Add here what changes were made in this pull request and if possible provide links showcasing the changes. -->
+
+
+  Screenshots of the change:
```

Comment:
Don't think that there will be screenshots often since there is not really an UI. Maybe also here add a link to the contributing page